### DISCUSSION TITLE
Why? After upgrading bootstrap and jszip, Trivy still reported the error that I should upgrade them.

### DISCUSSION THREAD

**Discussion by 2020AD** [aquasecurity/trivy#4314] [Q&A]
### Question

I scanned my project code with Trivy and it showed vulnerabilities in bootstrap and  jszip, prompting me to upgrade the version. However, after upgrading bootstrap and  jszip, Trivy still reported this error! It's really frustrating!
My Trivy version is Trivy@0.28.1
![image](https://github.com/aquasecurity/trivy/assets/38805891/7db58146-add2-472a-a05d-341ed38abee7)
![image](https://github.com/aquasecurity/trivy/assets/38805891/c9038084-1c29-4ed4-8b78-819a66c744b6)

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

**Comment by DmitriyLewen** [comment]
Hello @2020AD 
Thanks for your report!

First, i recommend that you update Trivy.` v0.28.1` is quite old.
After upgrading, you can use `--format json --list-all-pkgs` flags to get list of all found packages with path to package files.

Regards, Dmitriy
